packages/apputils-extension/src/index.ts

```diff
@@ -160,13 +160,7 @@ const themePlugin: JupyterLabPlugin<IThemeManager> = {
     let when = app.started;
     let manager = new ThemeManager({ baseUrl,  settingRegistry, host, when });
     let disposable = splash.show();
-    manager.ready.then(() => {
-      setTimeout(() => {
-        disposable.dispose();
-      }, 2500);
-    }, () => {
-      disposable.dispose();
-    });
+    manager.ready.then(() => disposable.dispose(), () => disposable.dispose());
```

Comment:
Since this notation:
```
() => disposable.dispose()
```
has the semantic meaning of:
```
() => { return disposable.dispose(); }
```
... we've been adopting the convention that we should only use the concise format if we actually *do* care about the return value. If there is no return value, or if we plan to ignore it, it'd be less ambiguous to use:
```
() => { disposable.dispose(); }
```

Reply:
makes sense, thanks.